lib/govuk_request_id.rb

```diff
@@ -0,0 +1,13 @@
+class GovukRequestId
+  def self.insert(body)
+    return body unless body.present?
+
+    doc = Nokogiri::HTML::DocumentFragment.parse(body)
```

Comment:
Fair enough. I was thinking on Govdelivery, or any automatic tool/sw within the validation loop, deciding to remove HTML comments 😃